extension/src/badge-service.js

```diff
@@ -0,0 +1,38 @@
+const DefaultsService = require('./defaults-service.js');
+
+const BadgeService = {
+	render(text, color, title) {
+		window.chrome.browserAction.setBadgeText({text});
+		window.chrome.browserAction.setBadgeBackgroundColor({color});
+		window.chrome.browserAction.setTitle({title});
+	},
+
+	renderCount(count) {
```

Comment:
Just export these methods individually, like:

```js
exports.renderCount = count => {
  ...
};
```

Then there's no need to use `this`.